hack-lint-test/action.yml

```diff
@@ -32,9 +32,10 @@ runs:
         echo "::endgroup::"
     - name: Lint
       shell: bash
+      # nightly lint errors are most likely just an hhast schema version issue
+      if: inputs.hhvm != "nightly"
```

Comment:
Does this work? I didn't see the `if` field in the spec: https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/metadata-syntax-for-github-actions#runs-for-composite-run-steps-actions

I don't see a good way to test, I guess we just have to commit and revert if it doesn't work.

I'm also not sure how this could be surfaced in the GitHub UI because the whole composite action is only ever surfaced as a single step in the workflow...

Reply:
huh, I was assuming it was the same as normal step syntax, but you're right, it's not there... I guess this is 'test in prod' time :/